client/src/components/layout/NewReviewForm.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react"
+import { useParams, Redirect } from "react-router-dom"
+import translateServerErrors from "../../services/translateServerErrors"
+import ErrorList from "./ErrorList"
+
+const NewReviewForm = (props) => {
+    const { id } = useParams()
+    
+    const [newReview, setNewReview] = useState({
+        userId: props.user.id,
+        destinationId: id,
+        title: "",
+        rating: "",
+        content: ""
+    })
+    const [errors, setErrors] = useState([])
+    const [shouldRedirect, setShouldRedirect] = useState(false)
+
+    const addNewReview = async () => {
+        try {
+            const response = await fetch(`/api/v1/destinations/${newReview.destinationId}/reviews`, 
```

Comment:
the destination id is included in the params of the URL meaning it should not be included in the body of data from the form